Sources/PostgREST/PostgrestBuilder.swift

```diff
@@ -51,9 +52,9 @@ public class PostgrestBuilder: @unchecked Sendable {
 
   /// Set a HTTP header for the request.
   @discardableResult
-  public func setHeader(name: String, value: String) -> Self {
+  public func setHeader(name: HTTPField.Name, value: String) -> Self {
```

Comment:
This is breaking change, lets keep the name as a String and create the HTTPField.Name internally.

Reply:
HTTPField.Name.init?(_ name: String) is optional init.
Can we use force unwrap?

```swift
@discardableResult
public func setHeader(name: String, value: String) -> Self {
  return self.setHeader(
    name: HTTPField.Name(name)!,
    value: value
  )
}
```

Reply:
yeah, force unwrap here is fine.

Reply:
ok, I. added.